web/json_options.go

```diff
@@ -75,6 +75,7 @@ func defaultJSONOptions() *jsonOptions {
 			{"01-02-2006", false},
 			{"01/02/2006", false},
 			{"01/02/06", false},
+			{"sgnl-unix-ns-timestamp", false},
```

Comment:
Let's stick to a naming convention similar to that of `time` pkg. I apologize for the repeated renaming.
```suggestion
			{"SGNLUnixNano", false},
```